lib/src/tileset/tile.dart

```diff
@@ -56,7 +56,7 @@ class Tile {
               .toList() ??
           [],
       image: parser.getSingleChildOrNullAs('image', TiledImage.parse),
-      objectGroup: parser.getSingleChildOrNullAs('objectGroup', Layer.parse),
+      objectGroup: parser.getSingleChildOrNullAs('objectgroup', Layer.parse),
```

Comment:
note: important to make sure the this works with json as well.
looking at [the docs[(https://doc.mapeditor.org/en/stable/reference/json-map-format/) it does appear so, but we dont have any json example with that field. would be easy to export you fixture map as json and add a test for that too?

Reply:
@luanpotter, I didn't get a chance to update this today, but I can add a test to include json before completing the PR tomorrow. It looks like it should work, and not too much work to include another test.

Reply:
@spydon did you want me to update the version number still, or just complete the PR with check failures, and you'll merge into your PR?

Reply:
@spydon did you want me to update the version number still. Or did you want me to leave the PR with check failures, and you'll merge into your PR with the updated version?

Reply:
I updated this branch to use the new cicd structure and now everything passes. We'd be really happy if you did a PR for the other issues that you found too! :)